account_journal_security/models/account_journal.py

```diff
@@ -106,11 +106,16 @@ def _search(self, domain, offset=0, limit=None, order=None, access_rights_uid=No
         cualquier lugar que se use un campo related a algo del diario
         """
         user = self.env.user
+        if limit == 1:
```

Comment:
@rov-adhoc  lo dejaria adentro del otro if  si soy superuser no deberia modificar el domain. if not self.env.is_superuser():
    if limit == 1: